### ISSUE TITLE
Chinese garbled(中文乱码),The Chinese description of the field in the mysql database is garbled,

### ISSUE THREAD

**Issue by fanhl** [linkedin/datahub#3157]
![image](https://user-images.githubusercontent.com/5433869/130738034-9758ce2c-62ed-4361-9925-0b0a5d4bf326.png)

**Comment by SakuraAxy** [comment]
I also encountered this problem and tried adding source.optons.encoding: utf-8  
But it doesn't work and problems still happen

**Comment by Light310** [comment]
Looks like cyrillic also broken

**Comment by dexter-mh-lee** [comment]
We recently updated init.sql to take in UTF8, but it doesn't convert existing ones. In the meantime can you try running 

```ALTER TABLE metadata_aspect_v2 CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;```

In the MySQL db?

**Comment by 1ambda** [comment]
Hi, after upgrading to 0.8.11 w/ Postgres, encountered the same problem. 
- dataset name, description, user name, ...

Datahub encoding is UTF-8.

```
SELECT pg_encoding_to_char(encoding) FROM pg_database WHERE datname = 'datahub_prod';

> UTF-8
```

And created database using `ko_KR.UTF-8'.

```
CREATE DATABASE datahub_prod OWNER xxx ENCODING 'UTF8' LC_COLLATE = 'C' LC_CTYPE = 'ko_KR.UTF-8' TEMPLATE template0;
```

**Comment by dexter-mh-lee** [comment]
Just to clarify. Do you see it correctly added to the MySQL/Postgres DB, but see bad decoding in the UI? or is it incorrectly ingested in the first place? @itc180016 @NightCat000 @1ambda

**Comment by dexter-mh-lee** [comment]
Hi everyone. We identified the issue. We will fix it asap and cut a release

**Comment by 1ambda** [comment]
Hi, @dexter-mh-lee 

In my case, We can see the text in DB and ES clearly in the original encoding, but Seems UI or API doesn't handle them correctly.

**Comment by NightCat000** [comment]
I already have solved the problem,if you want to how to solve,please click folowing:
https://blog.nowcoder.net/n/df53c7e3699d4828bcc1084f7e11bd36

**Comment by Echelon77** [comment]
> I already have solved the problem,if you want to know how to solve,please clicking folowing:
> https://blog.nowcoder.net/n/df53c7e3699d4828bcc1084f7e11bd36

It doesn't work for me. Maybe the reason is that the header of frontend reponse not set utf8, it responsed this: 'text/plain; charset=ISO-8859-1'

**Comment by NightCat000** [comment]
> > I already have solved the problem,if you want to know how to solve,please clicking folowing:
> > https://blog.nowcoder.net/n/df53c7e3699d4828bcc1084f7e11bd36
> 
> It doesn't work for me. Maybe the reason is that the header of frontend reponse not set utf8, it responsed this: 'text/plain; charset=ISO-8859-1'

I don’t know clearly why you are ,maybe you need to build new datahub after fixed it 。

**Comment by Echelon77** [comment]
> > > I already have solved the problem,if you want to know how to solve,please clicking folowing:
> > > https://blog.nowcoder.net/n/df53c7e3699d4828bcc1084f7e11bd36
> > 
> > 
> > It doesn't work for me. Maybe the reason is that the header of frontend reponse not set utf8, it responsed this: 'text/plain; charset=ISO-8859-1'
> 
> I don’t know clearly why you are ,maybe you need to build new datahub after fixed it 。

Yes，I pull the latest code and rebuild a new docker image, now it's solved.

**Comment by shirshanka** [comment]
This has been fixed via #3168 and is now available on head as @Echelon77 mentioned. 

We will cut a release (0.8.12) that contains this fix by end of this week. 
Please re-open if that doesn't resolve it.

**Comment by 1ambda** [comment]
It's fixed after upgrading to 0.8.12 :) Thanks.
